### PR TITLE
Xtokens snippet and learning pages links

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/382/files
https://github.com/PureStake/moonbeam-docs/pull/387

Some changes to Randomness page will be included when the Randomness pages get translated. 

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: builders/pallets-precompiles/pallets/staking.md [hunk 1]
```diff
@@ -10,9 +10,11 @@ keywords: staking, substrate, pallet, moonbeam, polkadot
 
 ## 概览 {: #introduction }
 
-Moonbeam通过[平行链staking](https://github.com/PureStake/moonbeam/tree/master/pallets/parachain-staking/src){target=_blank} pallet使用Delegated Proof of Stake（委托权益证明）系统，允许Token持有者（委托人）以准确表示他们希望支持的候选人（收集人）以及对收集人的质押量。此项平行链staking pallet的设计使得委托人和候选人（收集人）之间强制共享链上的风险/奖励。
+Moonbeam使用委托权益证明(DPoS)共识，该共识根据收集人在网络中的总质押数额决定哪些收集人有资格生产区块。有关质押的一般信息，例如术语、质押变量等，请参阅[质押](/learn/features/staking){target=_blank}页面。
 
-平行链staking pallet的一些功能也可以通过staking预编译实现。此预编译是一个Solidity接口，使您通过以太坊API执行质押动作。更多信息，请点击[Staking Precompile](/builders/pallets-precompiles/precompiles/staking){target=_blank}教程。
+DPOS共识允许Token持有者（委托人）以准确表示他们希望支持的候选人（收集人）以及对收集人的质押量。此项平行链质押pallet的设计使得委托人和候选人（收集人）之间强制共享链上的风险/奖励。
```
Comment (eshaben) [inline builders/pallets-precompiles/pallets/staking.md:15]: This looks like it is missing a link.

English version:
```
The DPoS system is powered by the [parachain staking](https://github.com/PureStake/moonbeam/tree/master/pallets/parachain-staking/src){target=_blank} pallet, allowing token holders (delegators) to express exactly which collator candidates they would like to support and with what quantity of stake. The design of the parachain staking pallet is such that it enforces shared risk/reward on chain between delegators and candidates.
```

**Review by eshaben — CHANGES_REQUESTED** [review]
just one thing!

**Comment by hyd628** [comment]
Thanks @eshaben , incorporated the feedback.

**Review by eshaben — APPROVED** [review]
Sweet thanks! LGTM ✨

**Pull request merged**